CalibPPS/ESProducers/plugins/CTPPSOpticalFunctionsESSource.cc

```diff
@@ -63,7 +63,7 @@ CTPPSOpticalFunctionsESSource::CTPPSOpticalFunctionsESSource(const edm::Paramete
     std::vector<FileInfo> fileInfo;
     for (const auto &pset : entry_pset.getParameter<std::vector<edm::ParameterSet>>("opticalFunctions")) {
       const double &xangle = pset.getParameter<double>("xangle");
-      const std::string &fileName = pset.getParameter<edm::FileInPath>("fileName").fullPath();
+      const std::string fileName = pset.getParameter<edm::FileInPath>("fileName").fullPath();
```

Comment:
@iarspider , what about this? why not remove the above and just use these directly in `fileInfo`?